scripts/acceptance.sh

```diff
@@ -2,6 +2,8 @@
 
 set -ex
 
+yarn webpack
```

Comment:
@eessex - had to add this here because we need a webpack-generated stats file to coordinate bundle splitting in our pages 😢